google-cloud-redis/src/test/java/com/google/cloud/redis/v1beta1/it/ITSystemTest.java

```diff
@@ -67,6 +70,8 @@ public static void setUp() throws Exception {
                 .build());
     CloudRedisSettings cloudRedisSettings = cloudRedisSettingsBuilder.build();
     client = CloudRedisClient.create(cloudRedisSettings);
+    /* Clean up old instances that were not deleted. */
+    cleanUpOldInstances();
```

Comment:
can't this call `v1.cleanUpOldInstances()` rather than duplicating the code?

Reply:
Used the same version of Instances as the test, in the cleanup for consistency. Guess it doesn't matter for cleanup. I will remove the duplication.